### DISCUSSION TITLE
Ran npm install -g zwave-js-ui to upgrade and backup directory wiped

### DISCUSSION THREAD

**Discussion by downtownsj1973** [zwave-js/zwave-js-ui#3763] [Q&A] [answered]
Well, SSIA, I ran the command npm install -g zwave-js-ui to upgrade to the latest version and I lost my store. I had backups scheduled but the backups directory appears to only contain a single entry from a recent backup. All the devices are seen but their data is zilch. I've tried searching the filesystem but I'm not seeming to find an obvious moved backup directory before the install.  Did I screw up the upgrade?  is there any chance of an on-sytem backup?  I had backed up other data directories but did not have a system wide backup

**Comment by kpine** [comment, marked as answer]
Here are the docs for your install method: https://zwave-js.github.io/zwave-js-ui/#/getting-started/other-methods?id=npm

There's a section with required setup:

![image](https://github.com/zwave-js/zwave-js-ui/assets/791794/3f109a3f-8720-47ed-bb26-af27970f62a7)

Setting those environment variables will put your storage directory outside of the default global directory. Right before that there's a highlighted warning:

![image](https://github.com/zwave-js/zwave-js-ui/assets/791794/124401f1-eb45-49a8-8263-3db0c831eca2)

When you run `npm install -g` a second time it seems to wipe the entire node_modules directory, which is where your store directory would have been located. On my Alpine linux container that would be `/usr/local/lib/node_modules/zwave-js-ui`. Sorry to say, but it looks like your settings are lost.

**Comment by robertsLando** [comment]
@downtownsj1973 I'm sorry for that but unfortunately this is well highlighted in docs as @kpine has shown in comment above :(

**Comment by downtownsj1973** [comment]
Absolutely mine to own:) thank you both for taking the time to help me be
better prepared the 2nd go round!
